files/walle.py

```diff
@@ -783,6 +815,32 @@ def get_database_with_password() -> RunningJobDatabase:
     )
 
 
+def post_slack_msg(message):
+    """Post a message to Slack."""
+    key = os.getenv("SLACK_API_TOKEN")
+    channel_id = os.getenv("SLACK_CHANNEL_ID")
+
+    if not all([key, channel_id]):
+        logger.warning(
+            "Slack notifications cannot be sent. Make sure your"
+            " playbook defines required vars:"
+            " [walle_slack_api_token, walle_slack_channel_id]."
+        )
+        return
```

Comment:
you probably did not use `get_str_from_env_or_error()` because you don't want it to raise an exception here, right?

Reply:
Yes, I guess if someone runs with `--kill --slack-alerts` it should keep running with `--kill`. Either way, all they get is an error/warning message in the log file right?